code/game/objects/items/scanners.dm

```diff
@@ -265,18 +268,27 @@ REAGENT SCANNER
 		var/current_organ = list(
 			"name" = organ.name,
 			"status" = organ.organ_status == ORGAN_BRUISED ? "Bruised" : "Broken",
-			"damage" = organ.damage
+			"damage" = organ.damage,
+			"effects" = organ.damage_effects,
 		)
 		damaged_organs += list(current_organ)
 	data["damaged_organs"] = damaged_organs
 
+	#define SYNTHETIC_REVIVE (patient.health >= patient.get_death_threshold())
+	#define ORGANIC_REVIVE (patient.health + patient.getOxyLoss() + (DEFIBRILLATOR_HEALING_TIMES_SKILL(user.skills.getRating(SKILL_MEDICAL))) >= patient.get_death_threshold())
```

Comment:
why are these defines?
Also the way you're using these is kinda messy. You're doing both of these revive checks no matter what, then checking if the patient is synth or meatboi repeatedly.

It would probably easier to just make a var to check if the patient is synth/meat, then just ref that var as needed.
Then have one revivable var which just varies based on the synth/meat var, and ref that as needed.